### ISSUE TITLE
ImportError: No module named 'recommonmark'

### ISSUE THREAD

**Issue by pitrou** [rtfd/readthedocs.org#1941]
This is what I get when trying a conda build now:
https://readthedocs.org/projects/-pitrou-llvmlite/builds/3647980/

I don't understand what happens, since RTD seems to use a conf.py file that has nothing to do with my project's.

```
 Running Sphinx v1.3.1
making output directory...

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/-pitrou-llvmlite/conda/rtd_conda/lib/python3.5/site-packages/Sphinx-1.3.1-py3.5.egg/sphinx/cmdline.py", line 244, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/-pitrou-llvmlite/conda/rtd_conda/lib/python3.5/site-packages/Sphinx-1.3.1-py3.5.egg/sphinx/application.py", line 126, in __init__
    confoverrides or {}, self.tags)
  File "/home/docs/checkouts/readthedocs.org/user_builds/-pitrou-llvmlite/conda/rtd_conda/lib/python3.5/site-packages/Sphinx-1.3.1-py3.5.egg/sphinx/config.py", line 277, in __init__
    execfile_(filename, config)
  File "/home/docs/checkouts/readthedocs.org/user_builds/-pitrou-llvmlite/conda/rtd_conda/lib/python3.5/site-packages/Sphinx-1.3.1-py3.5.egg/sphinx/util/pycompat.py", line 128, in execfile_
    exec_(code, _globals)
  File "conf.py", line 329, in <module>
    from recommonmark.parser import CommonMarkParser
ImportError: No module named 'recommonmark'

Exception occurred:
  File "conf.py", line 329, in <module>
    from recommonmark.parser import CommonMarkParser
ImportError: No module named 'recommonmark'
The full traceback has been saved in /tmp/sphinx-err-v95yzalf.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

**Comment by ericholscher** [comment]
This is the same issue as #1901 -- are you specifying a specific python version in your conda file, but not in your readthedocs.yml?

**Comment by pitrou** [comment]
Yes. Why should I do it in both? I don't remember this being documented at the time.

**Comment by ericholscher** [comment]
I just opened a PR to fix #1901 -- closing this, and hopefully that will fix the issues for you.
